examples/multiple-google-maps/app.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {FunctionComponent, useState, useCallback} from 'react';
+import {GoogleMapsProvider} from '@ubilabs/google-maps-react-hooks';
+
+import MapCanvas from './components/map-canvas/map-canvas';
+
+import {GOOGLE_MAPS_API_KEY} from '../constants';
+
+import './main.module.css';
+
+const basicMapOptions = {
+  zoom: 10,
+  disableDefaultUI: true,
+  zoomControl: true,
+  zoomControlOptions: {
+    position: 3 // Right top
+  }
+};
+
+// The Google Maps API parameters must be the same for all `GoogleMapsProvider` components!
+const googleMapsAPIParameters = {
+  language: 'it',
+  region: 'IT'
+};
+
+const App: FunctionComponent<Record<string, unknown>> = () => {
+  const [hamburgMapContainer, setHamburgMapContainer] =
+    useState<HTMLDivElement | null>(null);
+  const hamburgMapRef = useCallback(
+    (node: React.SetStateAction<HTMLDivElement | null>) => {
+      node && setHamburgMapContainer(node);
+    },
+    []
+  );
+  const hamburgMapOptions = {
+    ...basicMapOptions,
+    center: {lat: 53.551086, lng: 9.993682}
+  };
+
+  const [munichMapContainer, setMunichMapContainer] =
+    useState<HTMLDivElement | null>(null);
+  const munichMapRef = useCallback(
+    (node: React.SetStateAction<HTMLDivElement | null>) => {
+      node && setMunichMapContainer(node);
+    },
+    []
+  );
+  const munichMapOptions = {
+    ...basicMapOptions,
+    center: {lat: 48.137154, lng: 11.576124}
+  };
+
+  const [sanFranciscoMapContainer, setSanFranciscoMapContainer] =
+    useState<HTMLDivElement | null>(null);
+  const sanFranciscoMapRef = useCallback(
+    (node: React.SetStateAction<HTMLDivElement | null>) => {
+      node && setSanFranciscoMapContainer(node);
+    },
+    []
+  );
+  const sanFranciscoMapOptions = {
+    ...basicMapOptions,
+    center: {lat: 37.773972, lng: -122.431297}
+  };
+
+  return (
+    <div id="grid">
+      <GoogleMapsProvider
+        googleMapsAPIKey={GOOGLE_MAPS_API_KEY}
```

Comment:
`googleMapsAPIKey={GOOGLE_MAPS_API_KEY}`

Adding this multiple times, still gives me nightmares :D But okay, for now. 

Can we maybe add a comment somewhere here that it has to be added again but will not load multiple times? (I know it is also in the README below, but some hint here would be great. Maybe also in this example README.

Reply:
I moved the `googleMapsAPIKey` to the parameters object. I think it should be clearer now.

Reply:
Yes, perfect